.travis.yml

```diff
@@ -7,37 +7,40 @@ cache:
         - vendor
         - $HOME/.composer/cache/files
 
-php:
-    - 5.4
-    - 5.5
-    - 5.6
-    - 7.0
-    - 7.1
-    - 7.2
-    - 7.3
-
 env:
     - TWIG_EXT=no
+    - DRUPAL=no
 
 before_install:
-    # turn off XDebug
     - phpenv config-rm xdebug.ini || return 0
 
 install:
     - travis_retry composer install
+    - wget https://get.symfony.com/cli/installer -O - | bash
```

Comment:
why do we need that on all jobs ? `./drupal_test.sh` already does that

Reply:
a left-over, removed